examples/all-clusters-app/esp32/main/component.mk

```diff
@@ -44,6 +44,7 @@ COMPONENT_SRCDIRS :=
 
 COMPONENT_EXTRA_INCLUDES := $(PROJECT_PATH)/third_party/connectedhomeip/src/app/util                                   \
                             $(PROJECT_PATH)/third_party/connectedhomeip/src/app/server                                 \
+                            $(PROJECT_PATH)/third_party/connectedhomeip/src/app/clusters                               \
```

Comment:
Why this change?

Reply:
yeah looks like I didn't need this after all. Fixing.